### ISSUE TITLE
Your system folder path does not appear to be set correctly. Please open the following file and correct this: index.php

### ISSUE THREAD

**Issue by objecttothis** [jekkos/opensourcepos#1480]
### Issue / Question / Bug

Before submitting an issue please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [x] I'm reporting an issue of an unmodified OSPOS installation
- [x] I checked open and closed issues database and no similar issue was already discussed (please make sure you searched!)
- [x] I read the README, WHATS_NEW and UPGRADE
- [x] I read the FAQ (https://github.com/jekkos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP has php-gd, php-intl, sockets and etc. installed)
- [x] I read the wiki
- [x] I ran any database upgrade scripts (e.g. database/2.4_to_3.0.sql), and migrating function
- [x] I'm aware the latest master could be a development version and therefore not stable
- [x] I know the version of OSPOS and git commit hash (check the footer of your OSPOS), the name and version of OS, Web server, PHP and MySQL and will add them to my issue report

### Installation information

- OSPOS version is: 3.1.0
- OSPOS git commit hash is: 
- PHP version is:   (e.g. 5.5, 5.6, 7.0, 7.1) 5.6.31
- MySQL or MariaDB version is:   (e.g. MySQL 5.5, MySQL 5.6, MySQL 5.7, MariaDB 10.0, MariaDB 10.1, MariaDB 10.2) MySQL 5.7.19
- OS and version is:   (e.g. CentOS 6.9, Ubuntu 16.4, Windows 10) FreeBSD 11.0
- WebServer is:   (e.g. Apache 2.2, Apache 2.4, Nginx 1.12, Nginx 1.13) Apache 2.4.17
- (If applicable) Installation package for the LAMP/LEMP stack is:   (e.g. WAMP, XAMPP) FAMP

### Expected behaviour
Navigating to [BASE_DIR]/public/ would render the login screen.

### Actual behaviour
Error message "Your system folder path does not appear to be set correctly. Please open the following file and correct this: index.php" displays on the screen.

### Steps to reproduce the issue
It appears that several other people are experiencing this same error running various different installs [#1447](https://github.com/jekkos/opensourcepos/issues/1447)
- My file permissions are set to 644 for all files and 755 to all directories except [BASE_DIR]/application/logs/ which is 775 so that Apache/PHP can generate logs.
- logs are turned on in the config file but no logs are generated.
- Firebug shows that the error is generating an HTTP 503 "Service Unavailable" error.
- No PHP or MySQL logs are generated.
- I've uploaded a redacted version of the phpinfo() output.
- Same issue on HTTPS and HTTP connections.
- Same issue with mod_security, open_basedir, disable_functions and suhosin disabled.
- As far as I can tell I have all of the required modules installed and enabled.
- It appears that other people have reported this with XAMPP and WAMP installs but mine is a FAMP stack, so I'm not convinced that it's only the install.
[phpinfo.txt](https://github.com/jekkos/opensourcepos/files/1231233/phpinfo.txt)

**Comment by odiea** [comment]
Unless you are a developer and have all the needed programs to install and run the current github repository  the following is the location for the new zip files.
Local install

Dowload the latest stable release from github or unstable build from bintray
Create/locate a new mysql database to install open source point of sale into
Execute the file database/database.sql to create the tables needed
unzip and upload Open Source Point of Sale files to web server
Modify application/config/database.php and modify credentials if needed to connect to your database
Modify application/config/config.php encryption key with your own
Go to your point of sale install public dir via the browser
LOGIN using
username: admin
password: pointofsale
Enjoy
Oops an issue? Please make sure you read the FAQ, wiki page and you checked open and closed issue on GitHub. PHP display_errors is disabled by default. Create an application/config/.env file from the .env.example to enable it in a development environment.

**Comment by RamkrishnaMondal** [comment]
See #1447

**Comment by objecttothis** [comment]
@odiea did you mean to provide a link to something? You pasted the instructions for local install but referenced the repository in your message. Please note that I just downloaded 3.1.0 from the download link. If there is another download location that allows me to download a version with this issue resolved then please provide it. I am a developer but do not have time now to negotiate setting up GitHub in this environment.

**Comment by daN4cat** [comment]
Check the read me on the Github ospos front page, it contains the link to traybin for the zip download.

**Comment by objecttothis** [comment]
downloading and doing a fresh install of opensourcepos.20170816205819.HEAD.767b0d to see if the issue is resolved in that build.  I'll report back.

**Comment by objecttothis** [comment]
Confirmed. The opensourcepos.20170816205819.HEAD.767b0d build does not have this error.  Now I will resume my 2.0.2 to current migration with this build.  That ought to be fun.

**Comment by daN4cat** [comment]
Just to be clear latest build it's closed to the release one and looks stable. However it could still be that we need to fix a bug or two.

**Comment by guillermohaad** [comment]
Hi @objecttothis , download this versión opensourcepos.20170816205819.HEAD.767b0d

![image](https://user-images.githubusercontent.com/12921805/29435878-85018848-836e-11e7-9a40-09c8b57b651c.png)

But when I go in, he tells me:

An Error Was Encountered

Unable to load the requested language file: language/spanish/form_validation_lang.php

**Comment by odiea** [comment]
Is this a new pattern that is developing.
Unable to load the requested language file: language/indonesian/form_validation_lang.php #1458

**Comment by guillermohaad** [comment]
thank @odiea , I understand there is no solution yet?

**Comment by jekkos** [comment]
Ok I guess a mistake in build setup, had to add language before packaging.. updated travis config and will check in latest zip if ok

**Comment by daN4cat** [comment]
@guillermohaad try again in 5/10 min taking the last build from bintray.

**Comment by odiea** [comment]
I just downloaded and extracted the zip. Changed lang to es.
An Error Was Encountered

Unable to load the requested language file: language/spanish/form_validation_lang.php

**Comment by guillermohaad** [comment]
thank @daN4cat ,  
Again download the .zip opensourcepos.20170818082207.HEAD.97334b.zip ,  I will install only modifying the database.php with the credentials and follows the same error: Unable to load the requested language file: language / spanish / form_validation_lang.php
Note: I have not changed the default language as it comes.

**Comment by daN4cat** [comment]
Is it fine now?

**Comment by odiea** [comment]
opensourcepos.20170818185344.master.c57679.zip fixed nothing.

**Comment by paooxide** [comment]
I was able to resolve this: If using linux OS (mine is debian based)

do the following;
- open a terminal in the project folder

then enter; 
    composer install

# this should install the code igniter vendor folder and update dependencies. make sure internet connection is available as you may be required to update some php packages.

**Comment by wangxum168** [comment]
I had the same issue.
Please download "opensourcepos.20170909085348.3.1.1.76a002.zip",
Instead of "Source code (zip)"
The first one contains everything you need

**Comment by ErdoganYildirim** [comment]
Hello How I install OSPOS to any webhost I searched but I didn't find any example or document can you help me please !

**Comment by daN4cat** [comment]
@ErdoganYildirim, have you checked the wiki: https://github.com/opensourcepos/opensourcepos/wiki ?

**Comment by mckaygerhard** [comment]
again a response based on their own setup @daN4cat

@ErdoganYildirim ospos cannot install in "any" webhost, only on those that provided certain specifications. detailed in "i dont know anymore wiki page" (was changed from my setup)

as example locally "localhost" setup always will be possible, due you can install php-openssl/php5-mcrypt and setup any apache2 extension/htaccess ...

but if you wants to setup at webhosting provider, may have some problems,, may providers have limited environment, as example ospos can install at 000webhost.com free account only if you have a older account, ifr not you must buy a very expensive..

also opos does not install using softaculus due same reason.

in conclusion if you  not have experience you must use the venenux iso that comes with all already installed and ready to install/setup just following instructions from the wiki

**Comment by ErdoganYildirim** [comment]
Hello How can I solve this problem I tried and tired.Thanks to for helpers...
  **Your system folder path does not appear to be set correctly. Please open the following file and correct this: index.php**

**Comment by mckaygerhard** [comment]
@ErdoganYildirim what procedure do you use to react to that point?

**Comment by daN4cat** [comment]
@ErdoganYildirim have you downloaded the full installation from an official release?

**Comment by adesurahman32** [comment]
Your system folder path does not appear to be set correctly. Please open the following file and correct this: index.php for me now.. please help me..  version 3.3.1

**Comment by dennistsc** [comment]
> Your system folder path does not appear to be set correctly. Please open the following file and correct this: index.php for me now.. please help me.. version 3.3.1

i have the same problem. do you solve it yet? please help 
ubuntu 20.04
ospos version 3.3.6

**Comment by odiea** [comment]
Download the zip file under Install. You have to build the version that you are using.

**Comment by adityaharist** [comment]
Same issues, try to follow step by step on install.md but still the same issues on it. Anyone help, pls?

**Comment by odiea** [comment]
https://github.com/opensourcepos/opensourcepos/releases/download/3.3.8/opensourcepos.3.3.8.ffe492.zip
